File: verve patch/base/Applications/CSymphony_0_1/csSamples/SourceOut/CodeObjectCreateExpression.cpp
#pragma once 

#using <mscorlib.dll>

namespace MyNameSpace
{
    
    using namespace System;
    public __gc class Type1;
    
    
    public __gc class Type1
    {
        
        public: virtual System::Void MyMethod();
    };
    
    
    inline System::Void Type1::MyMethod()
    {
        System::DateTime();
    };
}

File: verve patch/base/Applications/CSymphony_0_1/vbSamples/CodeObjectCreateExpression.vb
Option Strict Off
Option Explicit On

Imports System
Imports System.CodeDom
Imports System.CodeDom.Compiler
Imports System.Collections
Imports CSymphony.Lib

Namespace CSymphony.CSy
	Public Class CSyUnit
		 Inherits GraphUnit
		Overrides Public Sub Build()
			ns("MyNameSpace")

			declClass(TypePublic, "Type1")
			declMethod(MemPublic, "System.Void", "MyMethod")
			enter (newObj("System.DateTime") )

		End Sub
	End Class
End Namespace
			